platform/ios/Integration Tests/MGLShapeSourceTests.m

```diff
@@ -122,5 +122,42 @@ - (void)testSettingShapeSourceToNilInRegionIsChanging {
     [self waitForExpectations:@[expectation] timeout:1.0];
 }
 
+- (void)testShapeSourceWithLineDistanceMetrics {
```

Comment:
Is this test supposed to be part of this PR?

Reply:
Yes, test verifies that MGLFeature is correctly converted to core type and line-metrics properties are populated.

Reply:
OK, perhaps we just need to update the PR title - as it seems this test is the bulk of the change?